packages/fxa-auth-server/lib/routes/utils/oauth.js

```diff
@@ -78,12 +83,13 @@ module.exports = {
         uid: credentials.uid,
       };
 
-      const account = await db.account(credentials.uid);
-      await mailer.sendNewDeviceLoginEmail(
-        account.emails,
-        account,
-        emailOptions
-      );
+      if (!isNewAccount) {
```

Comment:
micro-nit: putting `const geoData` and friends inside the `if (!isNewAccount)` block might increase the readability of this slightly, since the use of the `isNewAccount` is closer to the point where you calculate it so it's more obvious what it's going to be for.